EN: EP3 Dialogue and Minor Omnibus EP1 (Nearly done)

SorryLorry opened the pull request:
Please go ahead and make any changes if necessary. Definitely needs some grammar check.

SorryLorry committed:
EN: Ep3 Xiao Lobby-NPC Dialogue

alama committed:
Update np_010851.csv

alama committed:
Update np_010851.csv

DabirSA commented on Story/Episode 3/np_010851.csv:
```diff
@@ -0,0 +1,53 @@
+11_voice_NPC_5085_104#0,"""About ARKS... Let's start from the Beginning of Arks."""
+text01002#0,"""The name \""ARKS\"" was given to the team that investigates the various planets in the Universe."""
+text01003#0,"""The purpose of ARKS is not simply conducting investigations. They are also responsible for annihilating Darkers or any enemies corrupted by Darkers."""
```
"Darkers, or any enemies corrupted by Darkers."

DabirSA commented on Story/Episode 3/np_010851.csv:
```diff
@@ -0,0 +1,53 @@
+11_voice_NPC_5085_104#0,"""About ARKS... Let's start from the Beginning of Arks."""
+text01002#0,"""The name \""ARKS\"" was given to the team that investigates the various planets in the Universe."""
+text01003#0,"""The purpose of ARKS is not simply conducting investigations. They are also responsible for annihilating Darkers or any enemies corrupted by Darkers."""
+text01004#0,"""ARKS was first created by a Civilization known as the Photoners."""
```
"a civilization known as the Photoners."

DabirSA commented on Story/Episode 3/np_010851.csv:
```diff
@@ -0,0 +1,53 @@
+text01002#0,"""The name \""ARKS\"" was given to the team that investigates the various planets in the Universe."""
+text01003#0,"""The purpose of ARKS is not simply conducting investigations. They are also responsible for annihilating Darkers or any enemies corrupted by Darkers."""
+text01004#0,"""ARKS was first created by a Civilization known as the Photoners."""
+text01005#0,"""No... ARKS was created by a Being revered as a God by Photoners known as \""Xion\"". """
```
"a being revered as a God by Photoners, known as \\""Xion\\""."

DabirSA commented on Story/Episode 3/np_010851.csv:
```diff
@@ -0,0 +1,53 @@
+text01003#0,"""The purpose of ARKS is not simply conducting investigations. They are also responsible for annihilating Darkers or any enemies corrupted by Darkers."""
+text01004#0,"""ARKS was first created by a Civilization known as the Photoners."""
+text01005#0,"""No... ARKS was created by a Being revered as a God by Photoners known as \""Xion\"". """
+text01006#0,"""All for the purpose of defeating Darkers, and act as the Guardians of the Universe"""
```
"and to act as the guardians of the universe,"

DabirSA commented on Story/Episode 3/np_010851.csv:
```diff
@@ -0,0 +1,53 @@
+text01005#0,"""No... ARKS was created by a Being revered as a God by Photoners known as \""Xion\"". """
+text01006#0,"""All for the purpose of defeating Darkers, and act as the Guardians of the Universe"""
+text01007#0,"""Over the years, the objectives of ARKS seemed to have veered off course. Somehow, I managed to guide it back. """
+11_voice_NPC_5085_100#0,"""Exploring the Universe, Investigate Planets. Encountering any Darkers, fight and defeat them. That is ARKS."""
```
"Explore the universe, investigate planets. When encountering Darkers, fight and defeat them."

DabirSA commented on Story/Episode 3/np_010851.csv:
```diff
@@ -0,0 +1,53 @@
+text01006#0,"""All for the purpose of defeating Darkers, and act as the Guardians of the Universe"""
+text01007#0,"""Over the years, the objectives of ARKS seemed to have veered off course. Somehow, I managed to guide it back. """
+11_voice_NPC_5085_100#0,"""Exploring the Universe, Investigate Planets. Encountering any Darkers, fight and defeat them. That is ARKS."""
+11_voice_NPC_5085_106#0,"""About the Oracle huh? Oracle is basically an enormous fleet of Ships."""
```
"About Oracle, huh?"

DabirSA commented on Story/Episode 3/np_010851.csv:
```diff
@@ -0,0 +1,53 @@
+text02005#0,"""And the ship at the heart of the fleet, the \""Mothership\"", houses the core of my existence."""
+11_voice_NPC_5085_108#0,"""Previously, Xion was the core of the Mothership. However, we lost her to [Loser]. """
+text02007#0,"""Since then, I have succeeded Xion, assuming control of the whole fleet."""
+text02008#0,"""That sounds confusing huh.. To put it simply, Oracle is under my control."""
```
"That sounds confusing, huh." Watch your number of full stops.

DabirSA commented on Story/Episode 3/np_010851.csv:
```diff
@@ -0,0 +1,53 @@
+text02008#0,"""That sounds confusing huh.. To put it simply, Oracle is under my control."""
+text02009#0,"""Though it might get lonely without Xion... I have to push on."""
+11_voice_NPC_5085_100#1,"""For the future of Oracle. For ARKS. For you and me."""
+text03001#0,"""About Darkers... They spread throughout the Universe. An enemy of the Universe, an enemy of every living being."""
```
"the universe. An enemy of the universe, an enemy of all life."

DabirSA commented on Story/Episode 3/np_010851.csv:
```diff
@@ -0,0 +1,53 @@
+text03001#0,"""About Darkers... They spread throughout the Universe. An enemy of the Universe, an enemy of every living being."""
+text03002#0,"""Darkers consume all in their path. Every civilization, every living thing."""
+text03003#0,"""Only ARKS can permanently defeat Darkers with the use of Photons."""
+text03004#0,"""Even if someone is able to defeat the Darkers without the use of Photons, the residual fragments will eventually corrupt him. """
```
"eventually corrupt them." Watch that trailing space.

DabirSA commented on Story/Episode 3/np_010851.csv:
```diff
@@ -0,0 +1,53 @@
+11_voice_NPC_5085_104#1,"""Other means of defeating them may exist, if you consider the endless possibilities of the Universe."""
+text03009#0,"""For now, only ARKS is able to exterminate the Darkers. Which is why we are travelling across the Universe."""
+11_voice_NPC_5085_104#2,"""Dark Falz... To be honest, I do not have much information regarding them... So this will be short."""
+text04002#0,"""Currently, there is 5 confirmed existence of Dark Falz. [Elder], [Apprentice], [Loser], [Double] and lastly, [Persona]"""
```
"Currently, there are 5 confirmed Dark Falz."

DabirSA commented on Story/Episode 3/np_010851.csv:
```diff
@@ -0,0 +1,53 @@
+11_voice_NPC_5085_104#2,"""Dark Falz... To be honest, I do not have much information regarding them... So this will be short."""
+text04002#0,"""Currently, there is 5 confirmed existence of Dark Falz. [Elder], [Apprentice], [Loser], [Double] and lastly, [Persona]"""
+text04003#0,"""I can't say much about their characteristics... Well, you can simply think of them as the Commanders of Darkers."""
+text04004#0,"""[Elder] and [Apprentice] is more active than others in attacking ARKS."""
```
"[Elder] and [Apprentice] are"

DabirSA commented on Story/Episode 3/np_010851.csv:
```diff
@@ -0,0 +1,53 @@
+text04002#0,"""Currently, there is 5 confirmed existence of Dark Falz. [Elder], [Apprentice], [Loser], [Double] and lastly, [Persona]"""
+text04003#0,"""I can't say much about their characteristics... Well, you can simply think of them as the Commanders of Darkers."""
+text04004#0,"""[Elder] and [Apprentice] is more active than others in attacking ARKS."""
+text04005#0,"""[Loser] was defeated by Regius.. Due to [Double]'s power however, clones of [Loser] appear frequently."""
```
"[Loser] was defeated by Regius, but was devoured by [Double], and so clones of it appear frequently."

DabirSA commented on Story/Episode 3/np_010851.csv:
```diff
@@ -0,0 +1,53 @@
+text04005#0,"""[Loser] was defeated by Regius.. Due to [Double]'s power however, clones of [Loser] appear frequently."""
+text04006#0,"""As for [Persona]... Honestly, I dont know much."""
+text04007#0,"""Unless we are able to gather more information, there's nothing I can share. Sorry."""
+text04008#0,"""One thing is for sure, we will not have peace until Dark Falz is defeated."""
```
"until the Dark Falz are defeated."

DabirSA commented on Story/Episode 3/np_010851.csv:
```diff
@@ -0,0 +1,53 @@
+text04007#0,"""Unless we are able to gather more information, there's nothing I can share. Sorry."""
+text04008#0,"""One thing is for sure, we will not have peace until Dark Falz is defeated."""
+11_voice_NPC_5085_100#2,"""It's tough. That's why ARKS has to do it, only ARKS can do it."""
+11_voice_NPC_5085_100#3,"""Matter Board... I can tell you what I understand about it."""
```
"The Matter Board..."

DabirSA commented on Story/Episode 3/np_010851.csv:
```diff
@@ -0,0 +1,53 @@
+text04008#0,"""One thing is for sure, we will not have peace until Dark Falz is defeated."""
+11_voice_NPC_5085_100#2,"""It's tough. That's why ARKS has to do it, only ARKS can do it."""
+11_voice_NPC_5085_100#3,"""Matter Board... I can tell you what I understand about it."""
+text05002#0,"""As Xion was capable of calculating every possiblity in this Universe. She was able to calculate her desired outcome."""
```
"in this universe, she""

DabirSA commented on Story/Episode 3/np_010851.csv:
```diff
@@ -0,0 +1,53 @@
+11_voice_NPC_5085_100#2,"""It's tough. That's why ARKS has to do it, only ARKS can do it."""
+11_voice_NPC_5085_100#3,"""Matter Board... I can tell you what I understand about it."""
+text05002#0,"""As Xion was capable of calculating every possiblity in this Universe. She was able to calculate her desired outcome."""
+text05003#0,"""<%me>.<br>Matter Board was created for you to change the future."""
```
"The Matter Board was created to allow you"

DabirSA commented on Story/Episode 3/np_010851.csv:
```diff
@@ -0,0 +1,53 @@
+text05002#0,"""As Xion was capable of calculating every possiblity in this Universe. She was able to calculate her desired outcome."""
+text05003#0,"""<%me>.<br>Matter Board was created for you to change the future."""
+text05004#0,"""Besides making it possible to change the future, you could travel back in time to correct the past."""
+text05005#0,"""Matter Board allowed you to save people that you could not save before, or chancing upon encounters that were not supposed to happen"""
```
"The Matter Board"
"or chance upon encounters that were not supposed to happen."

DabirSA commented on Story/Episode 3/np_010851.csv:
```diff
@@ -0,0 +1,53 @@
+text05003#0,"""<%me>.<br>Matter Board was created for you to change the future."""
+text05004#0,"""Besides making it possible to change the future, you could travel back in time to correct the past."""
+text05005#0,"""Matter Board allowed you to save people that you could not save before, or chancing upon encounters that were not supposed to happen"""
+11_voice_NPC_5085_108#1,"""However, with Xion gone. I am unable to create a perfect Matter Board"""
```
"with Xion gone, I am"
"a perfect Matter Board."

DabirSA commented on Story/Episode 3/np_010851.csv:
```diff
@@ -0,0 +1,53 @@
+text05004#0,"""Besides making it possible to change the future, you could travel back in time to correct the past."""
+text05005#0,"""Matter Board allowed you to save people that you could not save before, or chancing upon encounters that were not supposed to happen"""
+11_voice_NPC_5085_108#1,"""However, with Xion gone. I am unable to create a perfect Matter Board"""
+text05007#0,"""My Matter Boards are simply an imitation of Xion's, weaker than Xion's."""
```
"are simply a weaker imitation of Xion's."

DabirSA commented on Story/Episode 3/np_010851.csv:
```diff
@@ -0,0 +1,53 @@
+text05005#0,"""Matter Board allowed you to save people that you could not save before, or chancing upon encounters that were not supposed to happen"""
+11_voice_NPC_5085_108#1,"""However, with Xion gone. I am unable to create a perfect Matter Board"""
+text05007#0,"""My Matter Boards are simply an imitation of Xion's, weaker than Xion's."""
+text05008#0,"""Although it is still capable of time travel and changing the future to a certain degree... Making significant changes to the past is simply out of the question."""
```
"Although they are still capable"

DabirSA commented on Story/Episode 3/np_010851.csv:
```diff
@@ -0,0 +1,53 @@
+text05008#0,"""Although it is still capable of time travel and changing the future to a certain degree... Making significant changes to the past is simply out of the question."""
+text05009#0,"""Because the Matter Board is the product of Xion's wishes, I am unable to fully grasp it."""
+text05010#0,"""However, I am able to understand and use Matter Boards left behind by Xion."""
+11_voice_NPC_5085_100#4,"""For now... Only I am able understand it. I should arrange it for people to understand it easier huh?"""
```
"am able to understand it."
"understand it more easily, huh?"

DabirSA reviewed (commented):
Grammar and punctuation, and a couple wording change suggestions that are probably down to taste. You have a lot of odd capitalisation on common nouns.

SorryLorry committed:
Ep3 Xiao Lobby-NPC Dialogue

Cleaned up grammar mistakes based on DabirSA's comments. Do share on any improvements in translation if possible.
Thanks in advance.

SorryLorry committed:
EN: Omnibus EP1 Ch.2 Alma's Summary

SorryLorry committed:
Adding breaks.

SorryLorry committed:
Update st_010230_om.csv

SorryLorry committed:
EN: EP3 Story Quest Unlocked

Minor TL

SorryLorry committed:
EN:Ep3 Matter Board cutscene

SorryLorry committed:
Update and rename Files/mb_030200.csv to Story/Episode 3/mb_030200.csv

SorryLorry committed:
EN:EP3 Chap4 Blue Skit

alama commented on Story/Episode 3/un_034090.csv:
```diff
@@ -0,0 +1,26 @@
+name01#0,"""？？？"""
+name02#0,"""？？？"""
+un_034090_0010#0,"""Ah, it's you.<br>Could you entertain Rela for me?"""
+un_034090_0020#0,"""Light is running away again. He gets frightened everytime Rela greets him with a roar. I'll drag him back here."""
```
`everytime` or `every time`?

alama committed:
Update un_034090.csv

alama committed:
Merge tag 'EN-v50101_rc_74' into SorryLorry

SorryLorry committed:
EN:EP3 Xiao: Confirming Current Situation

Half-done for now. As I am still progressing midway in the storyline, certain dialogue options are not open to me yet, nor have i progressed enough in the story to be fully able to flesh out the translation. Would be done in 2-3 days

SorryLorry committed:
Clean up of Grammar mistakes based on CircleCI

alama committed:
Merge remote-tracking branch 'upstream/EN' into EN

alama committed:
Merge remote-tracking branch 'upstream/EN' into EN

alama commented on Story/Episode 3/np_010852.csv:
```diff
@@ -0,0 +1,150 @@
+11_voice_NPC_5085_103#0,"""Luther said it was to understand Xion, but he just wanted her power."""
+text01005#0,"""However, half a year ago<br>With <%me>'s cooperation,<br>we managed to wrest control from Luther."""
+text01006#0,"""We discovered that Luther was<br>Dark Falz【Loser】. ARKS managed<br>to defeat him."""
+text01007#0,"""Oracle was released from the<br>Photonners... No, from Luther's<br>control."""
```
Why "Photonners"?

alama commented on Story/Episode 3/np_010852.csv:
```diff
@@ -0,0 +1,150 @@
+text03006#0,"""It would be best to assume that<br>【Persona】is equally unique as you,<br>possessing exceptional qualities."""
+text03007#0,"""However, I do have a hypothesis<br>but that requires further<br>investigations. I'll contact you<br>if there are further findings."""
+11_voice_NPC_5085_104#3,"""Analyzation of the data collected<br>from Wopal's Facility is<br>progressing smoothly. The data<br>has allowed us to know more<br>about Wopal."""
+text04002#0,"""Originally, Planet Wopal was a<br>designed as a resort for<br>Photonners."""
```
Photonners?

alama commented on Story/Episode 3/np_010852.csv:
```diff
@@ -0,0 +1,150 @@
+text04003#0,"""The ruins of an amusement park<br>and shiny structures can be<br>found."""
+text04004#0,"""However, something regarding<br>【Profound Darkness】caused the<br>resort to be abandoned."""
+text04005#0,"""As the Photonners travelled<br>through the universe, Luther<br>noticed the facility."""
+text04006#0,"""Not only was the facility<br>remodelled to do experiments,<br>it seems as if the planet was<br>remodelled as well."""
```
Photonners?

alama commented on Story/Episode 3/np_010852.csv:
```diff
@@ -0,0 +1,150 @@
+text05013#0,"""She was created to use an<br>imitation of Clarissa, which<br>pushed the limits of Techniques."""
+text05014#0,"""Her body age was set to be the<br>same as Sara, with certain<br>growth forced upon it. Her actual<br>age is actually 5-6 years old."""
+text05015#0,"""Well. Nothing can be done at<br>this point."""
+text05016#0,"""Both of them decided to live<br>their lives seperately as Sara<br>and Claris Claes."""
```
seperately or separately?

alama reviewed (commented):
You got some spelling mistakes

alama commented on Story/Episode 3/np_010852.csv:
```diff
@@ -65,16 +65,16 @@ text06007#0,"""Besides, there is a possibility<br>that this may occur to other<b
 text07008#0,"""While reapplying the seal,<br>【Double】's interference caused<br>a part of Magatsu to escape."""
-text07009#0,"""I am more concerned about how<br>【Double】is going to use Magatsu's<br>power than simply the fact that<br>a part of Magatsu escaped."""
-text07010#0,"""This isnt simply a problem for<br>Harkotan, but the universe as<br>well."""
+text07009#0,"""I am more concerned about how<br>【Double】is going to use Magatsus<br>power than simply the fact that<br>a part of Magatsu escaped."""
```
Should this be "Magatsus's power", not "Magatsus Power"?

SynthSy replied:
Magatsu's and 【】need to be replaced with [ and ]

SorryLorry committed:
Redone grammar (again)

alama committed:
Update PSO2.en_Enemies_Titans.dict

alama committed:
Update PSO2.en_Misc.dict

SorryLorry committed:
EP3-4 Matterboard A-2 Event

SorryLorry committed:
Lillipans not lilipans

SorryLorry committed:
EP3-4 Matterboard E-1 Event

SorryLorry committed:
Capitalization

SorryLorry committed:
Clarrisa Claes is now Claris Claes

SorryLorry committed:
Clarrisa evolved into Clarissa

SorryLorry committed:
More clarity regarding Claris.

SorryLorry committed:
Hopefully everything is Clarissa now

SorryLorry committed:
Adjusting Breaks of Dialogue

SorryLorry committed:
EP3-4 Matterboard A-5 Event

SorryLorry committed:
Adding "Afin's"

SynthSy commented on Story/Episode 3/un_034060.csv:
```diff
@@ -0,0 +1,23 @@
+un_034060_0150#0,"""Not at all. All of this is part<br>of my job, I do not mix personal<br>matters with official work."""
+un_034060_0160#0,"""By pointing out small errors,<br>they will be more meticulous.<br>This reduces the mistakes they<br>make in the long run."""
+un_034060_0170#0,"""ARKS is going through a shift in<br>structure. Which is why setting<br>things right is important."""
+un_034060_0180#0,"""Because of Luther, Regius is<br>still cast under a negative light.<br>The other members of the<br>Council of Six is out of the<br>question."""
```
Remove the line breaks here please.

SynthSy commented on Story/Episode 3/un_034080.csv:
```diff
@@ -0,0 +1,24 @@
+name02#0,"""？？？"""
+un_034080_0010#0,"""Ah,<%me>.<br>What a coincidence."""
+un_034080_0020#0,"""I wanted to interact with the Lillipans, thus I asked Fourier for help but...."""
+un_034080_0030#0,"""Lilili!\u3000Lilili!"""
```
The `\u3000` can be replaced with a space.

SynthSy commented on Story/Omnibus/st_010230_om.csv:
```diff
@@ -4,12 +4,12 @@ st_010230_0010#0,"""In all things, all possible outcomes cannot come to pass. On
-al_010030_0070#0,"""正直、怖いぐらいの能力だよ。<br>でもその力も、シオンが貴方を選んだ<br>理由の一つなんじゃないかな。"""
-al_010030_0080#0,"""おかげで、凶暴化するはずだった龍族が<br>正気に戻るという未来に書き換わり<br>アークスと龍族の間に、交流が生まれた。"""
-al_010030_0090#0,"""これが未来にどんな変化をもたらすか<br>まだわからないけど、シオンの目的には<br>必要不可欠な道筋、だったんだろうね。"""
+al_010030_0010#0,"""Darkers are a real pain aren't they.<br>Even after defeating them,<br>residual fragments corrupt organic<br>and inorganic alike,<br>causing them to go berserk."""
```
Don't preserve line breaks

SynthSy commented on Story/Episode 3/un_034040.csv:
```diff
@@ -0,0 +1,29 @@
+name02#0,"""？？？"""
+un_034040_0010#0,"""Yo <%me>!<br>You seem to be doing fine in ARKS."""
+un_034040_0020#0,"""I heard a lot of good things<br>about you."""
+un_034040_0030#0,"""I know that you are capable,<br>but you should take a break too."""
```
Again, don't preserve linebreaks.

SynthSy commented on Story/Episode 3/ra_034060.csv:
```diff
@@ -3,12 +3,12 @@ name02#0,"""???"""
 ra_034060_0010#0,"""I see...<br>The Dark Falz that appeared ten years ago;<br>[Took form of a child]."""
 ra_034060_0020#0,"""If one person confirmed that,<br>there's a possibility that their<br>memory's wrong.<br>But, if other people says the same,<br>then we're not mistaken."""
-ra_034060_0030#0,"""It seems like [Apprentice] wasn't the only Dark Falz<br>that appeared ten years ago, """
+ra_034060_0030#0,"""It seems like [Apprentice] wasn't the only Dark Falz that appeared<br>ten years ago, """
```
Don't preserve linebreaks here.

SynthSy commented on Story/Episode 3/np_010852.csv:
```diff
@@ -0,0 +1,150 @@
+11_voice_NPC_5085_104#0,"""The current situation?<br>It's a long story so I'll just<br>summarize it for you."""
```
This entire document has punctuation mistakes.

Also, do not preserve the linebreaks.

SynthSy reviewed (changes requested):
Overall, this PR needs a major re-work of the dialog and punctuation. Too many errors relating to sentence structure, as well as terms, and a complete disregard to formatting. These need to be corrected before this can be merged.

SorryLorry committed:
Xiao Dialogue (Not Complete)

SorryLorry committed:
Undo break edits of translated file

alama commented on Story/Episode 3/np_010852.csv:
```diff
@@ -1,92 +1,92 @@
+text02004#0,"""In order to stop herself from turning, she attempted to take her own life."""
+text02005#0,"""<%me> was present at that point, investigating the past. She was caught with you when I pulled you out of that situation."""
+text02006#0,"""You then stumbled upon her 10 years after, having lost both her memories and power."""
+text02007#0,"""If her loss was permanent, it wouldnt have made a difference. However, not only has she regained her powers, she managed to reunite with Clarissa."""
```
`wouldnt` or `wouldn't`

Kyle873 replied:
Wouldn't.

SorryLorry committed:
EP3-4 Matterboard E-1 Event.

SorryLorry committed:
Wouldn't

SorryLorry committed:
EP3-4 Matterboard A-5 Event

Cleared breaks, changed wording

SorryLorry committed:
EP3-4 Matterboard D-3 Event

Brackets left as it is to standardize dragonkin scenes with past translated files.

SorryLorry committed:
EP3-4 Matterboard A-2 Event

Cleared breaks. Rephrased

SorryLorry committed:
Xiao Dialogue (File2)

Cleared any breaks. Phrasing. Odd capitalization. Missing punctuation.

SorryLorry committed:
Omnibus EP1 Ch2, Alma's summary 

Fragments -> Darkness.
Reference drawn from one of the translated files about how Matoi was not able to contain the accumulated darkness.
Do enlighten me if there is a better term for it.

SorryLorry committed:
Ep3-5(?) Matterboard event

Slight changes after checking with ScienceJump on the file he translated. Changing on his behalf.
Unsure of exact event name as server is currently down

alama committed:
Merge tag 'EN-v50101_rc_89' into EN

SorryLorry committed:
Story quest unlock line

SorryLorry committed:
Update mb_030220.csv

missing "

SorryLorry committed:
Story unlock

SorryLorry committed:
New possibilities in story quest

"has been updated" for a lack of better terms.

SorryLorry committed:
Xiao Dialogue

Line 94: subspace for a lack of proper term.

Gonna need more time cause Xiao's way of speech is cryptic.

alama committed:
EN: fix misspelling

alama committed:
Merge remote-tracking branch 'upstream/EN' into SorryLorry

alama committed:
Merge tag 'EN-v50100_rc_113' into SorryLorry

alama committed:
Merge remote-tracking branch 'upstream/EN' into EN

alama committed:
EN: Dups lines

alama committed:
Merge remote-tracking branch 'upstream/EN' into EN

alama committed:
Merge remote-tracking branch 'upstream/EN' into EN

Aida-Enna commented:
Closed due to inactivity. @SorryLorry Feel free to comment on here if you're back and wish to pick up where you left off.
